Source/Core/VideoBackends/Software/SWRenderer.cpp

```diff
@@ -9,152 +9,47 @@
 
 #include "Common/CommonTypes.h"
 #include "Common/StringUtil.h"
-#include "Common/GL/GLInterfaceBase.h"
-#include "Common/GL/GLUtil.h"
 
+#include "Core/ConfigManager.h"
 #include "Core/Core.h"
+#include "Core/HW/Memmap.h"
 
-#include "VideoBackends/Software/RasterFont.h"
-#include "VideoBackends/Software/SWCommandProcessor.h"
+#include "VideoBackends/Software/EfbCopy.h"
+#include "VideoBackends/Software/SWOGLWindow.h"
 #include "VideoBackends/Software/SWRenderer.h"
-#include "VideoBackends/Software/SWStatistics.h"
 
+#include "VideoCommon/BoundingBox.h"
+#include "VideoCommon/Fifo.h"
 #include "VideoCommon/ImageWrite.h"
 #include "VideoCommon/OnScreenDisplay.h"
-
-static GLuint s_RenderTarget = 0;
-
-static GLint attr_pos = -1, attr_tex = -1;
-static GLint uni_tex = -1;
-static GLuint program;
+#include "VideoCommon/VideoConfig.h"
 
 static u8 *s_xfbColorTexture[2];
```

Comment:
Only one is used at all, and this should be moved to virtual XFB code. But this is in my opinion not needed for this PR.